packages/import-export/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "import-export",
+  "version": "2.9.1",
+  "main": "index.js",
```

Comment:
there is no `index.js` file in this package ?

Reply:
Is it the `/prepare-xlsx-1000.js` ?

Reply:
There is indeed no index.js file, and none that should be considered as an index.js file (the default file to use). I'll remove this entry.